algorithms/repository_rating.py

```diff
@@ -415,7 +415,7 @@ def calcRepoScore(self):
 
 
     
-accessToken = "03e5d817f468829fd9b3307f55de055461460c1a"
+accessToken = "<token-here>"
 headers = {"Authorization": "bearer "+ accessToken }
```

Comment:
would have liked if consumed it from SETTINGS (assuming we have a server_settings.py with the token), or implementing Github Auth and getting the token.

Reply:
This should be done in the subsequent versions perhaps.